shared/src/main/java/com/sap/psr/vulas/shared/json/model/ExemptionBug.java

```diff
@@ -89,17 +89,31 @@ public boolean isExempted(VulnerableDependency _vd) {
 		// Archive
 		if(is_exempted) {
 			// All
-			if(ALL.equals(this.digest)) { ; }
+			if(ALL.equals(this.library)) { ; }
 
-			// Digest
-			else if(this.digest.startsWith("dig:")) {
-				is_exempted = is_exempted && this.digest.substring(4).equals(_vd.getDep().getLib().getDigest());
+			// Package URL according to https://github.com/package-url/purl-spec
+			else if(this.library.startsWith("pkg:") && _vd.getDep().getLib().getLibraryId()!=null) {
+				try {
+					final LibraryId libid = _vd.getDep().getLib().getLibraryId();
+					final PackageURL purl = new PackageURL(this.library);
+					if(purl.getName()==null) {
+						log.error("The package URL [" + this.library + "] does not contain a name");
+					}
+					else {
+						is_exempted = is_exempted &&
+								(purl.getNamespace()==null || libid.getMvnGroup().equals(purl.getNamespace())) && // No purl.namespace || purl.namespace==libid.mvnGroup
```

Comment:
Why should we allow exemptions for library ids whose namespace (aka mvn group) is not specified?

Reply:
Package URLs for Python packages will not have a namespace